docs/docsite/rst/network/user_guide/platform_cnos.rst

```diff
@@ -0,0 +1,69 @@
+.. _cnos_platform_options:
+
+***************************************
+CNOS Platform Options
+***************************************
+
+CNOS supports Enable Mode (Privilege Escalation). This page offers details on how to use Enable Mode on CNOS in Ansible 2.7. 
+
+.. contents:: Topics
+
+Connections Available
+================================================================================
+
++---------------------------+-----------------------------------------------+
+|..                         | CLI                                           |
++===========================+===============================================+
+| **Protocol**              |  SSH                                          |
++---------------------------+-----------------------------------------------+
+| | **Credentials**         | | uses SSH keys / SSH-agent if present        |
+| |                         | | accepts ``-u myuser -k`` if using password  |
++---------------------------+-----------------------------------------------+
+| **Indirect Access**       | via a bastion (jump host)                     |
++---------------------------+-----------------------------------------------+
+| | **Connection Settings** | | ``ansible_connection: network_cli``         |
+| |                         | |                                             |
+| |                         | |                                             |
++---------------------------+-----------------------------------------------+
+| | **Enable Mode**         | | supported - use ``ansible_become: yes``     |
+| | (Privilege Escalation)  | | with ``ansible_become_method: enable``      |
+| |                         | | and ``ansible_become_pass:``                |
++---------------------------+-----------------------------------------------+
+| **Returned Data Format**  | ``stdout[0].``                                |
++---------------------------+-----------------------------------------------+
+
+For legacy playbooks, CNOS still supports ``ansible_connection: local``. We recommend modernizing to use ``ansible_connection: network_cli`` as soon as possible.
+
+Using CLI in Ansible 2.7
```

Comment:
Same issue, if the others agree, remove 2.7 here.

Reply:
All other vendors say ios, eos, junos all are putting the version 2.6 there. So I too followed the same. I feel it make sense to remove version from there. Whether to use network_cli or local is the confusion point for user, then he may refer the table in the platform_index.rst. I am waiting for Gundalow and/or privateip to respond.

Reply:
Yes, I'm thinking I need to go back in and remove 2.6 from all the others as well. But let's see if @gundalow, @privateip agree.

Reply:
I chatted with @gundalow in slack and he is okay with removing the version. I will go back and remove the 2.6 version from the other guides in https://github.com/ansible/ansible/issues/45630

Reply:
I will remove 2.7 from every where now. 
Thanks 
-Anil